prime-router/metadata/fhir_mapping/hl7/resource/Patient.yml

```diff
@@ -437,6 +437,32 @@ extension:
           vars:
             annotationText: NTE.3 *&, GeneralUtils.concatenateWithChar(annotationText, '\n')
 
+    - expressionType: nested
+      condition: tribalCitizenship NOT_NULL
+      generateList: true
+      vars:
+        tribalCitizenship: STRING_ALL, PID.39
+      constants:
+        hl7v2Name: "tribal-citizenship"
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: patient-citizenship
+        extension:
+          generateList: true
+          expressionType: nested
+          expressions:
+            - expressionType: nested
+              expressionsMap:
+                url:
+                  type: SYSTEM_URL
+                  value: code
```

Comment:
Take it or leave it, but I don't think all of these extensions strictly need to appear in the Mapping file and I think this could just be
```
type: STRING
value: 'code'
```